forte/pipeline.py

```diff
@@ -119,6 +119,28 @@ def __init__(self, resource: Optional[Resources] = None):
             self.resource = resource
 
         self.initialized: bool = False
+        self._check_type_consistency: bool = False
+
+    def enforce_consistency(self, enforce: bool = True):
+        r"""This function enforces a type consistency checking for
+        all the pipeline components. When this function is called
```

Comment:
Please explain the behavior at the beginning in a concise way, then another paragraph for the detailed description:

This function determines whether the pipeline will enforce the content expectations specified in each pipeline component. Each component will check whether the input pack contains the expected data via checking the meta-data, and throws a `forte.common.exception.ExpectedEntryNotFound` if the check fails.